### PR TITLE
Updates documentation for example with authentication flow

### PR DESCRIPTION
**Pull request by hayribakici**
This PR updates README on how to run the example with the authentication flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,10 @@ Run `pub run test` to run all of the tests in `test/spotify_test.dart`.
 ### Running example code
 Run `pub run example/example` to run the example code. You'll need to modify `example/example.dart` to use your Spotify client ID and secret.
 
+If you would like to run the example with user data, that is only available if you're authenticated, run `pub run example/example_auth.dart`. There, you are requested to
+enter your redirect url (see `redirectUri` above) and call a given url, with which you are _again_ requested to paste the redirected (see `responseUri` above) to access
+the user data. The output the available devices and currently playing song.
```
Comment (rinukkusu) [inline README.md:133]: I think this still needs some proof reading by you to make more sense :)

Comment (hayribakici) [reply README.md]: Yeah, you're right.

**Comment by ebarnsli** [comment]
just tagging on here for question but does this mean the refresh token auth stuff is working now 🤔

**Comment by rinukkusu** [comment]
@ebarnsli hi, yes - there is a working solution now - see readme :)

**Pull request merged**